lib/release/api.ts

```diff
@@ -194,31 +202,60 @@ async function getUser(
 	api: PinejsClientCore<unknown>,
 	id: number,
 ): Promise<models.UserModel> {
-	return models.get(api, 'user', id);
+	const user = await api
+		.get({
+			resource: 'user',
+			id,
+			options: { $select: 'id' },
+		} as const)
+		.catch(models.wrapResponseError);
+	if (user == null) {
+		throw new Error('Could not find user with id: ' + id);
+	}
+	return user;
 }
 
 async function getApplication(
 	api: PinejsClientCore<unknown>,
 	id: number,
 ): Promise<models.ApplicationModel> {
-	return models.get(api, 'application', id);
+	const app = await api
+		.get({
+			resource: 'application',
+			id,
+			options: { $select: 'id' },
+		} as const)
+		.catch(models.wrapResponseError);
+	if (app == null) {
```

Comment:
It could always be null but in the past it would have error'ed with something like `TypeError: cannot read 'id' of undefined` downstream